lib/opentok.js

```diff
@@ -62,8 +62,19 @@ function decodeSessionId(sessionId) {
  *
  * @param apiKey {String} Your OpenTok API key. (See your
  * <a href="https://tokbox.com/account">TokBox account page</a>.)
+ *
  * @param apiSecret {String} Your OpenTok API secret. (See your
  * <a href="https://tokbox.com/account">TokBox account page</a>.)
+ *
+ * @param env {Object} Configuration options for the client with the following options
+ * (all of which are optional)
```

Comment:
Let's mention that the parameter is optional, too:

```suggestion
 * @param env {Object} Configuration options for the client. This parameter is optional.
 * The object contains the following properties, all of which are optional:
```